src/scenes/home/resetPassword/requestToken/requestToken.js

```diff
@@ -27,13 +28,19 @@ class RequestToken extends Component {
     e.preventDefault();
     if (this.isFormValid()) {
       axios
-        .post(`${config.apiUrl}/users/passwords/reset`, {
+        .post(`${config.apiUrl}/users/passwords/forgot`, {
           email: this.state.email
         })
         .then(() => {
           this.setState({ success: true, error: null });
         })
-        .catch(() => {
+        .catch((err) => {
+          Raven.setUserContext({
```

Comment:
this shouldn't actually be necessary, as raven catches all exceptions, but i'll leave it cuz maybe context is helpful

Reply:
Context was the only reason I put this in here.